apps/namehashlabs.org/app/ens-referral-program/page.tsx

```diff
@@ -44,8 +44,10 @@ export default function Page() {
                   alt="hero"
                 />
               }
-              greenLabelText="Proposal pending"
-              buttonUrl="https://docs.google.com/document/d/1srqcho7PFyMBUDQTxxlH_eZqrt5x_EEB-PF2LfpYvIg/edit?usp=sharing"
+              calendarButtonText="Meet us at DevCon"
+              greenLabelText="Collecting Community Input"
+              buttonText="Share your voice"
+              buttonUrl="https://www.deform.cc/"
```

Comment:
@lightwalker-eth just a heads-up that I added the link https://www.deform.cc/ here